README.md

```diff
@@ -93,7 +96,6 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 |-----------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------|
 | otel.trace.exporter=jaeger-thrift-splunk | OTEL_TRACE_EXPORTER=jaeger-thrift-splunk              | Select the span exporter to use. `jaeger-thrift-splunk` is the default value.                                                                                          
 | otel.exporter.jaeger.endpoint     | OTEL_EXPORTER_JAEGER_ENDPOINT     | The Jaeger endpoint to connect to. Default is `http://localhost:9080/v1/trace`.
-| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance. Default is `unknown_service:java`, but any `service.name` provided via other means will supercede this value.
```

Comment:
Otel java autoconfiguration will not support this property?

Reply:
It doesn't support it right now, and seeing as in the current snapshot version there's no way to set service name other than resource attributes, it probably never will.

Reply:
we never added this to auto-configuration because it hasn't been in the spec since we created the auto-configuration module. That was always a "java-only" special that was waiting to be removed until the spec finally sorted out the required service name in the Resource.

Reply:
Let's not add anything not in spec. Or at least as little as possible :)